.github/workflows/on_push.yaml

```diff
@@ -0,0 +1,19 @@
+name: On Push
+
+# On push to a "special" branch, we:
+# * always publish to charmhub at latest/edge/branchname
+# * always run tests
+# where a "special" branch is one of main/master or track/**, as
+# by convention these branches are the source for a corresponding
+# charmhub edge channel.
```

Comment:
question: why is this workflow needed? Don't we already have a wf that does this on `main` and on `tracks`?

Reply:
that should definitely not be there, pushed by accident